unix-domain-socket/src/main/mima-filters/2.0.0-M1.backwards.excludes

```diff
@@ -0,0 +1,35 @@
+# PR #2012
```

Comment:
We normally put MiMa excludes in a separate file per PR for better organization.

```
<module>/src/main/mima-filters/<last-version>.backwards.excludes/<pr-or-issue>-<issue-number>-<description>.excludes
```

Reply:
Well, not _normally_, but it's something we've started doing since MiMa now supports it :)

Reply:
I've just gone through all of the sub projects and couldn't find an example. May I recommend that you separately create another PR reorganising all of the MiMa files to your new convention? I was expecting this to be a quick PR. ;-)

Reply:
Yes, that's a good point.  I don't think we've done this in Alpakka yet since adopting the new convention.  

You can find an example in Alpakka Kafka here: https://github.com/akka/alpakka-kafka/tree/master/core/src/main/mima-filters/1.1.0.backwards.excludes

All that needs to be done is to create `./unix-domain-socket/src/main/mima-filters/2.0.0-M1.backwards.excludes` as a directory and move your MiMa file into it and give it a filename like: `PR2012-unix-domain-socket-api-changes.excludes`.

Reply:
Unfortunately, I've no more time for this PR. I will adopt the convention going forward.

Reply:
Alright.  I pushed an update to your branch.  I'll merge it after the build is complete.

Reply:
Thanks mate.